lib/usb-hid-msc-lib.robot

```diff
@@ -0,0 +1,35 @@
+*** Keywords ***
+Mount Bootable ISO
+    [Documentation]    Mounts a bootable ISO as flash USB. Currently
+    ...    only the Qubes OS ISO seems to work for the platform.
+    Mount Image    ${PIKVM_IP}    Qubes-R4.2.0-rc1-x86_64.iso
+
+Navigate Through Some Menus
+    [Documentation]    Attempts to navigate And Enter some menus
+    ...    using the USB keyboard to verify that it works.
+    Press Key N Times    1    ${ENTER}
+    Read From Terminal Until    Standard Fran
+    Press Key N Times    3    ${ARROW_DOWN}
+    Press Key N Times    1    ${ESC}
+
+    Press Key N Times And Enter    1    ${ARROW_DOWN}
+    Read From Terminal Until    Admin Pass
+    Press Key N Times    1    ${ESC}
+
+    Press Key N Times And Enter    1    ${ARROW_DOWN}
+    Read From Terminal Until    Driver Health
+    Press Key N Times    4    ${ARROW_DOWN}
+    Press Key N Times    4    ${ARROW_UP}
+    Press Key N Times    1    ${ENTER}
+    Read From Terminal Until    Physical
+    Press Key N Times    2    ${ESC}
+
+    Press Key N Times And Enter    1    ${ARROW_DOWN}
+    Press Key N Times    4    ${ARROW_DOWN}
+    Press Key N Times    4    ${ARROW_UP}
+    Press Key N Times    1    ${ESC}
+
+Find The Boot USB
```

Comment:
This is exactly against what we need. 
We are adding keyword that works only for one particular case (where PiKVM) is connected, and what is more, there are already existing keywords offering this: https://github.com/Dasharo/open-source-firmware-validation/blob/main/keywords.robot#L2896

Reply:
Resolved in [recent commit](https://github.com/Dasharo/open-source-firmware-validation/commit/9a7f15c18a71110e706347fdaa24543ddfc08a90)